src/components/NetlifyForm/index.jsx

```diff
@@ -0,0 +1,91 @@
+/*
+ * NetlifyForm component consists of server related functionalities that is used posting data and also avoiding captchas.
+ * The base code that is used in both netlifyForm and index are implemented from https://github.com/jon-fm/react-ssg-netlify-forms-demo
+ */
+import React, { useState, useEffect } from "react";
+
+const NetlifyForm = ({
+  formName,
+  beforeSubmit,
+  afterSubmit,
+  contactFormValues,
+  children,
+}) => {
+  // Keep things well-hidden during runtime (and from bots. They are exposed for netlify build bots.
```

Comment:
Keep what hidden?

Reply:
It keeps hidden Honey form field, I also updated my comment so that it could understand better.